etl-job/Dockerfile

```diff
@@ -9,54 +9,30 @@ RUN apt-get -y --no-install-recommends install  \
            gcc \
            git \
            iputils-ping \
-           jq \
-           libmagic-dev \
-           libpq-dev \
-           postgresql-client \
            python3-dev \
            unzip \
            vim \
            automake \
            libtool \
-	   wget \
-           ca-certificates
+	   wget
 
 RUN rm -rf /var/lib/apt/lists/*
 
-
 # set the working directory in the container
 WORKDIR /root
 
-RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
-RUN unzip awscliv2.zip
-RUN ./aws/install
-RUN rm awscliv2.zip
-
 SHELL ["/bin/bash", "-c"]
 RUN python3 -m venv venv
 RUN source venv/bin/activate && pip install wheel
-RUN source venv/bin/activate && pip install yq
-
-RUN mkdir ~/.aws
 RUN mkdir ~/.gen3
 
 ENV PYTHONUNBUFFERED=1
 COPY ./requirements.txt /root
 RUN source venv/bin/activate && pip install -r requirements.txt
 
 # move these out of requirements.txt, so that builds are quicker when we change
-RUN source venv/bin/activate && pip install "iceberg_tools>=0.0.8rc3"
 RUN source venv/bin/activate && pip install "gen3_tracker==0.0.5rc13"
```

Comment:
should be >= 0.0.6 ?

Reply:
Unsure where iceberg tools is being used in this repo?

Reply:
Sorry the comment  should have been on gen3_tracker line

Reply:
Oh. yes gen3_tracker should be updated. Maybe make it 0.0.7rc2 when that branch gets approved an the release gets cut: https://github.com/ACED-IDP/gen3_util/tree/release/0.0.7rc2